README.md

```
@@ -1,6 +1,6 @@
 # SEC EDGAR Bulker
 
-A robust Python project for downloading and parsing SEC EDGAR filings with parallel processing capabilities.
+A robust Python project for downloading and parsing SEC EDGAR filings with parallel processing capabilities and more. 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)